test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+# Run helm test
+# Any arguments passed to this script will be passed to `helm test`.
+
+HELM_TEST_ARGS="${@:---cleanup}"  # cleanup test pod by default
+
+RELEASE_NAME="test-$(date -u +%Y%m%d-%H%M%S)"
+
+function finish() {
+  echo "> Deleting release $RELEASE_NAME"
+  helm del --purge $RELEASE_NAME
+}
+trap finish EXIT
+
+function main() {
+  echo "> Installing helm chart, waiting until app is ready..."
+  helm install --wait --name $RELEASE_NAME \
+    --set dataKey="$(docker run --rm cyberark/conjur data-key generate)" \
+    ./conjur-oss
+
+  echo "> Running helm tests with arguments: $HELM_TEST_ARGS"
+  helm test $HELM_TEST_ARGS $RELEASE_NAME
+}
+
+main
```

Comment:
Given how small this file is, I don't know if main() function is needed (vs running toplevel) but your call